Core/main.cpp

```diff
@@ -44,6 +51,54 @@ int main(int argc, char* argv[])
 	}
 
 	glfwTerminate();
+}
+
+
+
+void run_physics(Physics& physx) {
+	while (true) {
+		physx.Update();
+		std::this_thread::sleep_for(std::chrono::milliseconds(1000));
+	}
+}
+
+void run_ui(UI& ui) {
+	while (true) {
+		ui.Update();
+		std::this_thread::sleep_for(std::chrono::milliseconds(1000));
+	}
+}
+
+
+
+int main(int argc, char* argv[])
+{
+
+	std::cout << "Do you know what DK Stands for? Donkey Kong? Nah. Drift King." << std::endl;
+
+
+	// TODO: Make this a call to the Render Project
+	//run_glfw();
+
+	UI ui;
```

Comment:
An object was not actually created here

Reply:
From what @Tissure mentioned, they were planning to keep it here for now
Should I move it to the Render project?

Reply:
I mean that I don't think this actually creates the object Ui.

`UI ui = new UI(();` is necessary, otherwise the next lines operate on an uninitialized variable